avif/_api.py

```diff
@@ -40,6 +40,23 @@ def parse_path(self, path):
         _succeed(lib.avifDecoderParse(self._decoder))
         return self._decoder.image
 
+    def parse_data(self, data):
```

Comment:
Cool! So, usually with CFFI, you want to start to grow a "Python-native" API that looks more like what folks would expect out of Python than out of C.

I haven't yet done that -- `Decoder` is the very start of such an API. But yeah just wanted to mention that... in other words, at some point someone should be able to write something more like `for image in decoder.frames` rather than needing to manually loop over `n_frames`, as one does in C.

But we can improve that over time.